lib/class-vip-request-block.php

```diff
@@ -81,6 +81,21 @@ public static function ua( string $user_agent ) {
 		return false;
 	}
 
+	/**
+	 * Block by partial match of the user agent header
+	 *
+	 * @param string $user_agent_substring target user agent to be blocked.
+	 * @return void|bool
+	 */
+	public static function ua_partial_match( string $user_agent_substring ) {
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotValidated, WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___SERVER__HTTP_USER_AGENT__
+		if ( isset( $_SERVER['HTTP_USER_AGENT'] ) && str_contains( $_SERVER['HTTP_USER_AGENT'], $user_agent_substring ) ) {
```

Comment:
`str_contains()` appeared in PHP 8; we still need to be compatible with PHP 7.4. Could you please update the code to use `strpos()` or whatever is appropriate?

Thank you!